### PR TITLE
New Template - hp_procurve_show_interfaces

### PR DESCRIPTION
**Pull request by adraf82**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

hp_procurve_show_interfaces.textfsm

##### SUMMARY

I have added a new template for the 'show interfaces' command for a hp procurve device.

```

```

### CODE REVIEW

**Comment by adraf82** [comment]
Hi, has anyone been able to review my commits yet? Please can you advise if any amendments need to be made. This is my first pull request on GitHub. :-) Thanks

**Comment by jmcgill298** [comment]
Thanks Adraf82. The tests are failing, but I believe that if you rebase this with the current master, then they will likely pass. Please let us know if you need help with rebasing.

File: templates/index [hunk 1]
```diff
@@ -373,6 +373,7 @@ hp_comware_display_vlan_brief.textfsm, .*, hp_comware, di[[splay]] v[[lan]] b[[r
 hp_comware_display_clock.textfsm, .*, hp_comware, di[[splay]] clo[[ck]]
 hp_comware_display_arp.textfsm, .*, hp_comware, di[[splay]] a[[rp]]
 
+hp_procurve_show_interfaces.textfsm, .*, hp_procurve, sh[[ow]] int[[erfaces]]
```
Comment (jmcgill298) [inline templates/index:376]: Please follow the entry rules:

```
# Rules of Ordering:
#  - OS in alphbetical order
#  - Command in length other
#  - When Length is the same, use alphabetical order
#  - Keep space between OS's
#
```

**Review by jmcgill298 — CHANGES_REQUESTED** [review]

**Comment by adraf82** [comment]
Thanks for the reply Jacob. I have corrected the order for my template within the index file and have committed the change for merging.

I have rebased my repo and the test that was previously failing has passed.

Thanks

Adeel

Sent from my iPhone

On 20 Oct 2020, at 23:11, Jacob McGill <notifications@github.com> wrote:

﻿

Thanks Adraf82. The tests are failing, but I believe that if you rebase this with the current master, then they will likely pass. Please let us know if you need help with rebasing.

**Review by jmcgill298 — APPROVED** [review]

**Comment by jmcgill298** [comment]
Thanks @adraf82

**Pull request merged**